test/journeys/.eslintrc.js

```diff
@@ -8,6 +8,11 @@ module.exports = {
     $: true
   },
   rules: {
-    'no-console': 'off'
+    'no-console': 'off',
+    'prefer-arrow-callback': [
+      'error', {
+        allowNamedFunctions: true
```

Comment:
Needed to allow named mocha functions for `this.retries()`

Reply:
Good to know!